### ISSUE TITLE
Manage account: change of email to user jmunroe

### ISSUE THREAD

**Issue by jmunroe** [pypa/pypi-support#58]
I have an account "jmunroe" that is associated with the now non-existent email address "jmunroe@ualberta.ca". Could the email be changed to "jmunroe@mun.ca" so that I can reset the password and regain access to this account? Thank you.

**Comment by di** [comment]
Hi @jmunroe, sorry for the delay. Do you have any way to verify that you're the original owner of this account or of the `igwtools` project?

**Comment by jmunroe** [comment]
Hi @di . This igwtools package was written by me during my time as a PhD student at University of Alberta (my PhD thesis: https://era.library.ualberta.ca/items/ed6eceed-20ec-47a2-af5c-67e459c7281c ).  The software was developed 2006-2008 to analyze "internal gravity waves" for my PhD work.  I moved to Memorial University of Newfoundland ( faculty web page: https://www.mun.ca/physics/our_people/faculty/munroe.php showing that I did my PhD at the University of Alberta) where I continued to do research in internal gravity waves  and stratified fluids ( https://www.physics.mun.ca/~jmunroe/lab/ )

The code was developed with an internal Subversion server that has long since been retired so I don't have anything public to point you towards. From my own email records, I can see that my PyPi account associated with the email address jmunroe@ualberta.ca was originally registered Thursday, Apr 26, 2007.  

Is that sufficient?
